response_operations_ui/templates/secure-message/misc-inbox.html

```diff
@@ -0,0 +1,238 @@
+{% extends "layouts/base.html" %}
+{% from "components/search/_macro.njk" import onsSearch %}
+{% from "components/section-navigation/_macro.njk" import onsSectionNavigation %}
+{% from "components/table/_macro.njk" import onsTable %}
+{% from "components/input/_macro.njk" import onsInput %}
+
+{% set page_title = "Messages" %}
+{% set hide_breadcrumbs = true %}
+{% block main %}
+
+  {% with messages = get_flashed_messages() %}
+    {% if messages %}
+          {% call
+          onsPanel({
+              "type":"success",
+              "icon": "check",
+              "classes": "u-mb-s",
+              "bodyAttributes": {
+                 "data-qa": "success-body"
+              }
+          })
+      %}
+        {% for message in messages %}
+          <p id="flashed-message-{{ loop.index }}">{{ message }}</p>
+        {% endfor %}
+      {% endcall %}
+    {% endif %}
+  {% endwith %}
+  <div class="grid grid--flex grid--between">
+    <div class="grid__col"> 
+      {% if breadcrumbs %}
+        {% include 'layouts/configs/breadcrumbs.html' %}
+      {% endif %}
+      <h1 class="u-fs-l u-mb-xs" name="page-messages-title">FDI Messages
+        <span class="u-fs-r">(<a id="change-survey" href="{{ url_for('messages_bp.select_inbox') }}">Change inbox</a>)</span>
+      </h1>
+      {% if ru_ref_filter %}
+          <p id="clear_ru_ref" class="u-fs-s">Filtered by RU ref: {{ ru_ref_filter }}
+            <a href="{{ url_for('messages_bp.clear_filter', selected_survey=selected_survey, conversation_tab=conversation_tab) }}">Clear filter</a>
+          </p>
+      {% endif %}  
+    </div>
+    <div class="grid__col u-mt-s@s"> 
+      <form action="{{ url_for('messages_bp.view_technical_inbox', selected_survey=selected_survey, conversation_tab=conversation_tab, ru_ref_filter=ru_ref_filter, business_id_filter=business_id_filter, page=page, limit=limit) }}" method="post">
```

Comment:
the url_for needs to be different here

Reply:
yes.. missed it totally. Copy paste error. fixed it